### PR TITLE
Support CLOAD instruction

### PR DESCRIPTION
**Pull request by shermike**
- Fix bug with integer length in printf
- Support CLOAD instruction
- Enable several tracers simultaneously

### CODE REVIEW

File: lib/evmone/instructions.hpp [hunk 1]
```diff
@@ -1119,6 +1121,8 @@ inline DynStackResult printf(
     auto stack_pop = &stack_origin.top() - &handler.stack.top();
     assert(stack_pop < std::numeric_limits<int16_t>::max());
 
+    state.dyn_stack_pop = static_cast<unsigned>(stack_pop);
```
Comment (knazarov) [inline lib/evmone/instructions.hpp]: Nit: it looks a bit hacky to only set `dyn_stack_pop` in printf, given that the function returns `stack_pop`. Is it possible to set `dyn_stack_pop` somewhere on the outer scope?

Comment (shermike) [reply lib/evmone/instructions.hpp]: You're right, we can assign it in the common handler. Fixed, thanks.

File: lib/evmone/vm.cpp [hunk 2]
```diff
@@ -59,12 +59,12 @@ evmc_set_option_result set_option(evmc_vm* c_vm, char const* c_name,
         vm.add_tracer(create_instruction_tracer(std::clog));
         return EVMC_SET_OPTION_SUCCESS;
     }
-    else if (name == "trace-bin")
+    if (name == "trace-bin")
```
Comment (avm) [inline lib/evmone/vm.cpp]: In any given invocation `name` has only one value, so there does not seem to be a point in removing the `else`?

Comment (shermike) [reply lib/evmone/vm.cpp]: Yeah, my fault, it is already worked. Fixed, thanks.

**Review by knazarov — APPROVED** [review]

**Pull request merged**